astroquery/image_cutouts/first/__init__.py

```diff
@@ -17,7 +17,7 @@ class Conf(_config.ConfigNamespace):
     Configuration parameters for `astroquery.image_cutouts.first`.
     """
     server = _config.ConfigItem(
-        ['https://third.ucllnl.org/cgi-bin/firstcutout'],
+        ['https://sundog.stsci.edu/cgi-bin/firstcutout'],
```

Comment:
> (I tried to figure out how to tell users to modify the URL themselves, but that seems to require them to edit the configuration file. Run-time changes didn't work for me.)

FWIW, for IRSA I routinely do run-time changes to swap between the dev/test and ops servers. The difference I see is that I don't have the value in a list. According to the astropy.config docs if the value is a list the behaviour is totally different 🤯 

So, I would consider this as a bug for this module as we don't have mirrors here that we want to validate, but only one server with the option of maybe overriding the url.

I leave it up to you if you want to leave as is or change it to be not a list.

Reply:
OK, I did some more digging and this is a more widespread problem -- I opened https://github.com/astropy/astroquery/issues/3414 to propose we clean up all the modules that doesn't explicitly work with mirrors.

(And I'm afraid you might have inherited this from the template module itself which shows how to deal with mirrors)